pkg/storaged/filesystem/mismounting.jsx

```diff
@@ -178,9 +178,9 @@ export const MismountAlert = ({ warning, fstab_config, forced_options, backing_b
         else if (type == "mount-on-boot")
             return do_mount();
         else if (type == "no-mount-on-boot")
-            return do_unmount();
+            return do_unmount(old_dir);
         else if (type == "mounted-no-config")
-            return do_unmount();
+            return do_unmount(other);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19697-20231206-092935-a03118a2-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

Reply:
Uhhh, this seems wrong

Reply:
Locally:

```
+        console.log(`UMOUNT ${type} ${other} ${old_dir}`);
         if (type == "change-mount-on-boot")
```

```
-> ph_mouse("[data-test-card-title='ext4 filesystem'] button:contains('Unmount now'):not([disabled]):not([aria-disabled=true])","click",0,0,0,false,false,false,false)
> log: UMOUNT mounted-no-config /run/bar undefined
```